src/decision/location_selection.jl

```diff
@@ -153,7 +153,8 @@ function ranks_to_location_order(ranks::NamedDimsArray)
     end
     return location_orders
 end
-function ranks_to_location_order(ranks::NamedDimsArray, iv_type::String)
+function ranks_to_location_order(ranks::NamedDimsArray, 
+    iv_type::String)
```

Comment:
Hi Rose, is the JuliaFormatter doing this?

Reply:
No it's not, I did that trying to get it to work, is there anything else I have to do other than have the JuliaFormatter extension and the Julia formatter file?

Reply:
@Zapiano any advice here?

Reply:
@Rosejoycrocker If you press Ctrl+Shift+P and write "Settings" and in the Settings search bar write "format on save", can you confirm that the "Format on save" option is selected?

Reply:
Hi @Zapiano, just checked and format on save is on

Reply:
And you have a file named `.JuliaFormatter.toml` (starting with a dot) and the content of this file is

```
style = "blue"
```

?

Reply:
If all the above is true, one more thing: you said you have another formatter extension installed, right? One called "black formatter" or something? Have you tried disabling it?

Reply:
I have the .JuliaFormatter.toml file with style="blue". I have it in the ADRIA.jl folder, is this the correct place?

I don't have any other Julia formatters installed. Black formatter I use to format my Python code.

Reply:
Yes, I have my `.JuliaFormatter.toml` file inside Adria.jl folder...

Are you sure Black formatter is not interfering with Julia formatter (even thought it is only for python)? 

The one you have is this, right?

![image](https://github.com/open-AIMS/ADRIA.jl/assets/8040719/e87923d1-2704-4784-a966-ca2d004d19a5)

v.1.2.1? Enabled globally? I'm running out of ideas...

Reply:
I workled it out, I had format on save on but I seeded to change the mode to "modifications" :'D
Thanks @Zapiano !

Reply:
Awesome!